packages/web-app-preview/src/App.vue

```diff
@@ -153,6 +139,10 @@ export default defineComponent({
     const cachedFiles = ref<CachedFile[]>([])
     const folderLoaded = ref(false)
 
+    const mimeTypes = computed(() => {
+      return getMimeTypes(appsStore.externalAppConfig[appId]?.mimeTypes)
```

Comment:
Can't we use the applicationConfig prop for this?

Reply:
As discussed, we need this to be wrapped in `AppWrapper` first.